src/main/resources/config/application-prod.yml

```diff
@@ -14,6 +14,10 @@ spring:
       uri: ${SPRING_DATA_NEO4J_URI}
       password: ${SPRING_DATA_NEO4J_PASSWORD}
       username: ${SPRING_DATA_NEO4J_USERNAME}
+  datasource:
+    url: ${SPRING_DATA_NEO4j_JDBC_URI}
```

Comment:
What about making external configuration a bit easier by supplying this preconfigured pattern: `jdbc:neo4j:${SPRING_DATA_NEO4J_URI}`

We would need to set only one environment variable in production and get a consistent Neo4j configuration.

Reply:
Thank you. 👍  I changed the property as you suggested. The reason I used the dedicated environment variable is that I was not sure if mixture of simple string and environment variable placeholder would be resolved properly. But let's give it a try. I think it should work.